### ISSUE TITLE
Add missing name for test card close button

### ISSUE THREAD

**Issue by nathancrtr** [CDCgov/prime-simplereport#4018]
## Area or Component
Conduct Test

## Quick Summary
The "X" to close the test card is just marked "close", but that's not descriptive enough - it should more accurately be "close test", or better yet, "close test for [person]"

## Accessibility Violation
* `aria-name-missing-incorrect`
    * The element's accessible name is missing or is not appropriate for the element's purpose.
    * https://cdc-devtools.dequecloud.com/rule-help/en/aria-name-missing-incorrect

## Action Requested
Update aria label to be more descriptive - "close test", or even "close test for [person]"

## Source Code (optional)
https://github.com/CDCgov/prime-simplereport/blob/f11273c49f8e25f8a85449b510e6947a49d3457c/frontend/src/app/testQueue/QueueItem.tsx#L643-L650
